.github/workflows/ci-cache.yaml

```diff
@@ -1,11 +1,7 @@
 name: cache CI
 
 on:
-  workflow_dispatch: # Allow manual runs.
-  pull_request:
-    branches: [ 'main' ]
-  push:
-    branches: [ 'main' ]
```

Comment:
Are you going to restore these? Seems necessary. If so, another way is to ignore the build failure and force-submit this PR (after temporarily changing the repo setting, if necessary).

Reply:
the gcs cache is not in use. I was going to delete that code. But that gcs cache fails for the same reason that this update was previously failing.

Reply:
I will have an update to remove gcs cache completely.